example_updates/nominal_update/example-a-b-update.sh

```diff
@@ -0,0 +1,768 @@
+#!/bin/bash
```

Comment:
/usr/bin/env bash is more portable and will not fail if bash is installed elsewhere (e.g. /usr/local/bin/bash) but is in the $PATH, but disadvantage is that it would be a bit less secure as it's searching the PATH which could've been compromised.